build/bin/build-client-binary

```diff
@@ -22,29 +22,30 @@ gobuild_component() {
         echo skip build
         return
     fi
-
+    if [ ${component} = 'pmm-admin' -o ${component} = 'pmm-agent' ]; then
+        build_command="cd /tmp/go/src/${extract_path} &&  make release"
+    else
+        build_command="go build -o \${binary_dir}/${result_file} ./src/${extract_path}/${component_path}"
+    fi
     docker run --rm -v ${tmp_dir}:/home/builder/tmp ${rpmbuild_docker_image} sh -c "
         set -o errexit
         set -o xtrace
         export GOOS=${GOOS:-linux}
         export GOARCH=${GOARCH:-amd64}
         export CGO_ENABLED=0
+	export PMM_RELEASE_PATH=\${binary_dir}/bin
+	export PMM_RELEASE_VERSION=${pmm_version}
```

Comment:
Why we have `$` there, but `\$` on the previous line?

Reply:
it is needed to get correct path in docker:
with: /home/builder/
without: /home<local_user>